aop/aop-symbol-processor/src/main/kotlin/ru/tinkoff/kora/aop/symbol/processor/AopProcessor.kt

```diff
@@ -29,13 +30,13 @@ class AopProcessor(private val aspects: List<KoraAspect>, private val resolver:
 
         override fun constructorParam(type: KSType, annotations: List<AnnotationSpec>): String {
             return constructorParams.computeIfAbsent(ConstructorParamKey(type, annotations, resolver)) { key ->
-                this.computeFieldName(key.type.toString())!!
+                this.computeFieldName(key.type.makeNotNullable().toString())!!
```

Comment:
I think that we should pass type to that function and process it name and type params

Current changes don't fix problem with such types as `Foo<String?>`

Reply:
Generic type doesn't take part in name computing, this part is cut off when computing name so its is not a problem